src/main/resources/hudson/plugins/performance/PerformanceReportMap/index.jelly

```diff
@@ -19,9 +19,7 @@
          <j:forEach var="uriReport" items="${performanceReport.getUriListOrdered()}">
            <tr class="${h.ifThenElse(uriReport.failed,'red','')}">
              <td class="left">
-               <a href="./uriReport/${uriReport.encodeUriReport()}">
-               <st:out value="${uriReport.getShortUri()}" />
-               </a>
+               <st:out value="${uriReport.getStaplerUri()}" />
```

Comment:
I'm not sure this approach is right, because a wrapper hyperlink has been just removed. Maybe you should just keep HREF and st:out separately.